### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by CloverGit**

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -222,10 +238,7 @@ def config_account(self):
         print('* Username and password set successfully!')
 
     def detect_ip(self):
-        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        s.connect(('www.baidu.com', 80))
-        ip = s.getsockname()[0]
-        s.close()
+        ip = urllib2.urlopen('http://ip.42.pl/raw').readline()
```
Comment (CloverGit) [inline install.py]: This is a stable and free external resources

File: core/modules/server.py [hunk 2]
```diff
@@ -793,8 +806,7 @@ def supportfs(self):
 
     print('* LVM partitions:')
     for partition in diskinfo['lvm']['partitions']:
-        print('  - Partition name: %s (%d, %d)' % \)
-            (partition['name'], partition['major'], partition['minor'])
```
Comment (CloverGit) [inline core/modules/server.py]: It cause an error when I used the --dev arg

File: install.py (hunk 1 above)
Comment (doudoudzj) [reply install.py]: urModule urllib2, please compatible with python2 and python3

File: static/partials/main.html [hunk 3]
```diff
@@ -409,6 +421,6 @@ <h3>欢迎使用 InPanel ！</h3>
                     </tbody>
                 </table>
             </div>
-        </div>
+	</div>
```
Comment (doudoudzj) [inline static/partials/main.html]: Code indent, please replace tabs with spaces

File: core/modules/server.py (hunk 2 above)
Comment (doudoudzj) [reply core/modules/server.py]: Thank you for your contribution

**Comment by doudoudzj** [comment]
Thank you for your contribution, please optimize your code and submit the merge request again.

File: static/partials/main.html [hunk 4]
```diff
@@ -136,17 +142,23 @@ <h3>欢迎使用 InPanel ！</h3>
...
                                         <td ng-bind-html-unsafe="info['server.meminfo']['swap_used_rate'] | space.used"></td>
                                     </tr>
                                     <tr>
-                                        <td>剩余可用：</td>
+                                        <td>空闲空间：</td>
                                         <td class="text-right">{{ info['server.meminfo']['swap_free'] }}</td>
                                         <td></td>
                                         <td ng-bind-html-unsafe="info['server.meminfo']['swap_free_rate'] | space.free"></td>
                                     </tr>
+				    <tr>
```
Comment (doudoudzj) [inline static/partials/main.html]: please replace tabs with spaces

File: static/partials/main.html [hunk 5]
```diff
@@ -89,27 +89,33 @@ <h3>欢迎使用 InPanel ！</h3>
...
                                         <td ng-bind-html-unsafe="info['server.meminfo']['mem_used_rate'] | space.used"></td>
                                     </tr>
                                     <tr>
-                                        <td>剩余可用：</td>
+                                        <td>物理空闲：</td>
                                         <td class="text-right">{{ info['server.meminfo']['mem_free'] }}</td>
                                         <td></td>
                                         <td ng-bind-html-unsafe="info['server.meminfo']['mem_free_rate'] | space.free"></td>
                                     </tr>
+				    <tr>
```
Comment (doudoudzj) [inline static/partials/main.html]: please replace tabs with spaces

**Pull request closed without merging**